src/hooks/abilities/useAbilities.test.ts

```diff
@@ -1,22 +1,45 @@
 import { mockFunction, renderHook } from '@app/test-utils'
+// import { writeFileSync} from 'fs'
```

Comment:
remove

Reply:
I have a script that groups the results together to make the results easier to read at the end that uses the writeFileSync, which is why I commented it out instead of removing it.